docs/customizations/footer.md

```diff
@@ -0,0 +1,218 @@
+# Footer
+
+## Table of contents
+
+- [Interfaces](#interfaces)
```

Comment:
consider for the default section to add as reference the permalink for the specific block of code

Reply:
same concern for line references as below